core/build.gradle

```diff
@@ -181,16 +179,16 @@ dependencies {
             'org.json:json:20240303',
             'org.yaml:snakeyaml:2.2',
             'com.github.spullara.cli-parser:cli-parser:1.1.6',
-            'org.apache.httpcomponents:httpclient:4.5.14',
             'com.sun.mail:javax.mail:1.6.2',
             'com.amazonaws:aws-java-sdk-s3:1.12.770',
-            'com.adobe.xmp:xmpcore:6.1.11',
             'io.sentry:sentry-logback:7.5.0',
+            // For JSON logs
             'net.logstash.logback:logstash-logback-encoder:8.0',
+            // For PDF/A
+            'com.adobe.xmp:xmpcore:6.1.11',
     )
 
     implementation(
-            'org.apache.xmlgraphics:xmlgraphics-commons:2.9',
```

Comment:
but it is used by batik-codec. Sure we don't need it ? Or we don't have tests requiring it ?

Reply:
To be sure I roll back it :-), thanks

Reply:
Dependencies (cleaned):
```
+--- ar.com.fdvs:DynamicJasper:5.3.9
|    \--- org.apache.xmlgraphics:xmlgraphics-commons:2.6 -> 2.9 (*)

+--- org.geotools:gt-svg:30.2
|    +--- org.geotools:gt-render:30.2 (*)
|    +--- org.apache.xmlgraphics:batik-transcoder:1.17
|    |    +--- org.apache.xmlgraphics:batik-bridge:1.17
```